[Playstation] Fix build break after 252799@main

jigenzhou opened the pull request:
#### 0165256e096498ad0b027e563ff854158c4d702c
<pre>
[Playstation] Fix build break after 252799@main

<a href="https://bugs.webkit.org/show_bug.cgi?id=243223">https://bugs.webkit.org/show_bug.cgi?id=243223</a>

Unreviewed build fix.
Skip createTemporaryDirectory() for PlayStation since temporary directory is not supported on PlayStation.

Canonical link: <a href="https://commits.webkit.org/252953@main">https://commits.webkit.org/252953@main</a>
</pre>

basuke commented on Source/WTF/wtf/FileSystem.cpp:
```diff
@@ -834,7 +834,7 @@ String pathByAppendingComponents(StringView path, const Vector<StringView>& comp
-#if !OS(WINDOWS) && !PLATFORM(COCOA)
+#if !OS(WINDOWS) && !PLATFORM(COCOA) && !PLATFORM(PLAYSTATION)
 
 String createTemporaryDirectory()
```
Did you confirm the function is not used in PlayStation platform?

jigenzhou replied:
createTemporaryDirectory() is not currently in use in PlayStation.

jigenzhou committed:
[Playstation] Fix build break after 252799@main

https://bugs.webkit.org/show_bug.cgi?id=243223

Unreviewed build fix.
Skip createTemporaryDirectory() for PlayStation since temporary directory is not supported on PlayStation.

Canonical link: https://commits.webkit.org/252953@main
